airtel/driver/driver.go

```diff
@@ -0,0 +1,19 @@
+package driver
```

Comment:
Do we need this driver package for anything? Just remove it completely.

Reply:
okay